packages/worker/src/features.ts

```diff
@@ -0,0 +1,26 @@
+import env from "./environment"
+
+enum WorkerFeature {}
+
+const featureList: WorkerFeature[] = processFeatureList()
+
+function processFeatureList() {
```

Comment:
NAB: Might be worth putting this piece of feature flag logic into `backend-core` as it's shared